docs/development/extensions-core/lookups-cached-global.md

```diff
@@ -204,7 +204,8 @@ The remapping values for each globally cached lookup can be specified by a JSON
         "\"value\"]"
       ]
   },
-  "pollPeriod":"PT5M"
+  "pollPeriod":"PT5M",
+  "maxSize": 5242880
```

Comment:
I think it'd make more sense to express this as a percentage of available heap instead of a fixed byte limit, since the amount of available memory can differ across the various nodes where lookups might be loaded

Reply:
Good suggestion, fixed.